README.md

```diff
@@ -41,6 +41,7 @@ This plugin is compatible with CSI versions [v1.1.0](https://github.com/containe
 | v0.5.x (beta)                        | no            | no   | no   | yes  | yes  | yes   |
 | v0.6.x (beta)                        | no            | no   | no   | yes  | yes  | yes   |
 | v0.7.x (beta)                        | no            | no   | no   | no   | no   | yes   |
+| v1.0.x (ga)                          | no            | no   | no   | no   | no   | yes   |
```

Comment:
we recently added the snapshot sidecar. Should we make the minimum version 1.17?

Reply:
Done.